docs: Incorrect link to using components from creating components

MattLish opened the pull request:
`Using-components.md` doesn't exist.
I changed the link to `components.md` instead, which is where I assume the link was supposed to take me.


## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈
(I'm unable to sign the license agreement currently. I will keep retrying, otherwise feel free for someone else to just make this change)

- [x] `npm test` passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [x] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [x] Affected artifact templates in `packages/cli` were updated
- [x] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

dhmlau commented on docs/site/Creating-components.md:
```diff
@@ -6,7 +6,7 @@ sidebar: lb4_sidebar
 ---
 
-As explained in [Using Components](Using-components.md), a typical LoopBack
+As explained in [Using Components](components.md), a typical LoopBack
```
I believe you're referring to https://github.com/strongloop/loopback-next/blob/master/docs/site/Components.md. The file name here is case-sensitive. 

```suggestion
As explained in [Using Components](Components.md), a typical LoopBack
```

MattLish replied:
I probably am, yes. 
It was just a guess to where the docs were supposed to be taking me.
If that's the correct place, then I will change it.

dhmlau replied:
If that's the case, yes, please use `Components.md` instead. Thanks.

dhmlau commented:
@MattLish, could you please also sign the CLA? https://cla.strongloop.com/agreements/strongloop/loopback-next. Thanks.

MattLish commented:
@dhmlau I squashed down the commit (and included you name which GitHub included by default). Hope that's okay.
I've managed to signed the CLA now.

dhmlau commented:
@MattLish, one last thing, there's a code lint error (surprisingly with this small change). Could you please fix the linting error by running `npm run lint:fix` ? Thanks.

MattLish commented:
@dhmlau I'll be honest I didn't even run that because I didn't think such a small change would cause an issue, ha

I have linting errors on master too, so when I go to fix my branch it edits multiple files.
I've just added and committed the change to the file I edited only.
Squashed again too.

dhmlau commented:
@MattLish, thanks for fixing the linting error. Your PR has landed! 🎉
